src/main/java/frc/robot/RobotContainer.java

```diff
@@ -154,6 +155,7 @@ private void initCommands() {
     if (intakeSubsystem != null) {
       intakeForwardCommand = new IntakeForwardCommand(intakeSubsystem);
       intakeReverseCommand = new IntakeReverseCommand(intakeSubsystem);
+      Shuffleboard.addEventMarker("Reverse Intake Activated", EventImportance.kLow);
```

Comment:
This isn't a reverse intake event, just creating the instance of the IntakeReverseCommand class we'll use. Should probably have this called inside the initialize() method of that command instead

Reply:
Deleted and moved to the proper location